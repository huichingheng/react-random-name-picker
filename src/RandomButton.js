  import React from "react";

const RandomButton = props => {
  return (
    <div>
      <button onClick={() => props.handleClick()}>get lucky winner!</button>
    </div>
  );
};

export default RandomButton;
