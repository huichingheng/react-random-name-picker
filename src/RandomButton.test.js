import React from "react";
import RandomButton from "./RandomButton";
import ShallowRender from "react-test-renderer/shallow";

it("renders div with some text", () => {
  const renderer = new ShallowRender();
  renderer.render(<RandomButton />);
  const output = renderer.getRenderOutput();

  expect(output).toMatchSnapshot();
});

it("renders div with a button ", () => {});
