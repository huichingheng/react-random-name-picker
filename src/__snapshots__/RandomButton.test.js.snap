// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders div with some text 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    get lucky winner!
  </button>
</div>
`;
